lib/project/results-view.js

```diff
@@ -96,18 +96,28 @@ class ResultsView {
   }
 
   getRowHeight(resultRow) {
+    let baseHeight;
+
     if (resultRow instanceof LeadingContextRow) {
-      return this.contextRowHeight
+      baseHeight = this.contextRowHeight
     } else if (resultRow instanceof TrailingContextRow) {
-      return this.contextRowHeight
+      baseHeight = this.contextRowHeight
     } else if (resultRow instanceof ResultPathRow) {
-      return this.pathRowHeight
+      baseHeight = this.pathRowHeight
     } else if (resultRow instanceof MatchRow) {
-      return this.matchRowHeight
+      baseHeight = this.matchRowHeight
     }
+
+    return this.getHeightAdjustedToFontSize(baseHeight)
+  }
+
+  getHeightAdjustedToFontSize(baseHeight) {
+    const lineHeight = 2;
+    const baseFontSize = 12;
```

Comment:
12 was the default font size before my changes. I'm treating it as base size for height calculation because some of the existing stylings seems to depend on this value.